docs/src/proposals/fee_transaction_priority.md

```diff
@@ -165,46 +182,24 @@ task to schedule the next transactions.
 
 1. Once a BankingStage thread finishes processing a batch of transactions `completed_transactions_batch` ,
 it sends the `completed_transactions_batch` back to the scheduler via the same channel to signal of completion.
```

Comment:
Is it correct to think that transactions in Banking thread's buffer will be: 
1. processed successfully (it is no longer possible of AccountInUse), or
2. retried due to cost limits (block, account or account data size), or 
3. forwarded to next leader(s);

Case 1 results scheduling next transactions. How about case 2 and 3?

Reply:
On the second thought, if banking threads can feedback to Scheduler that "transaction A was not processed and dropped from banking thread buffer because it will not  exceed cost limit(s)", and Scheduler will handle that, then it probably makes sense to apply cost limit logic at banking thread (vs in Scheduler) in front of its buffer.